src/test/java/uk/gov/dhsc/htbhf/page/SendCodePage.java

```diff
@@ -4,14 +4,13 @@
 import org.openqa.selenium.support.ui.WebDriverWait;
 import uk.gov.dhsc.htbhf.page.component.RadioButton;
 
-import static uk.gov.dhsc.htbhf.page.component.RadioButton.EMAIL;
-import static uk.gov.dhsc.htbhf.page.component.RadioButton.TEXT;
-
 /**
  * Page object for the send code page
  */
 public class SendCodePage extends SubmittablePage {
 
+    private static final String TEXT = "text";
```

Comment:
This was from Matt's comment on the previous PR (didn't want to ask for re-approvals on that PR).